test/unit/model/serializer/test_serializer.py

```diff
@@ -51,16 +52,16 @@ def test_save_model(
         mock_log_figure,
         mock_log_model,
     ):
-
         model_type = "xgb"
         model = ModelCreator.create_model(model_type)
-        pj = TestData.get_prediction_job(pid=307)
+        pj, modelspecs = TestData.get_prediction_job(pid=307)
+        pj["id"] = "Default"
```

Comment:
So MLflow writes it to the default experiment since this prediction will not save all data needed.
changing the ID will is necessary to make sure we only save valid models in the experiment